modules/portworx-data-services/examples/simple/main.tf

```
@@ -0,0 +1,26 @@
+# TEMPLATE: This file was automatically generated with `generate_addon_structure.sh`
+# TEMPLATE: and should be modified as necessary
+# TEMPLATE:
+# TEMPLATE: All ".tf" files are parsed at once. There is no benefit to numerically prefixed filenames. Keep all resource definitions in "main.tf".
+# TEMPLATE:
+# TEMPLATE: When main.tf becomes unwieldy, consider submodules (https://www.terraform.io/docs/language/modules/develop/structure.html) 
+# TEMPLATE: and dependency inversion (https://www.terraform.io/docs/language/modules/develop/composition.html).
+#
+provider "equinix" {
+  auth_token = var.metal_auth_token
+}
+
+module "equinix_kubernetes_addons" {
+  source = "../../../../"
+
+  kubeconfig_local_path = var.kubeconfig_local_path
+
+  enable_portworx_data_services = true
+
+  portworx_data_services_config = {
+    pds_account_id = var.pds_account_id
+    pds_token      = var.pds_token
+    tenant_id      = var.tenant_id
```

the module main.tf file expects `pds_tenant_id`

```suggestion
    pds_tenant_id  = var.pds_tenant_id
```